stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Messages.kt

```diff
@@ -0,0 +1,195 @@
+package io.getstream.chat.docs.kotlin
+
+import android.util.Log
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.api.models.SearchMessagesRequest
+import io.getstream.chat.android.client.channel.ChannelClient
+import io.getstream.chat.android.client.errors.ChatError
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.client.models.Filters
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.Reaction
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.client.utils.FilterObject
+import io.getstream.chat.android.client.utils.ProgressCallback
+import io.getstream.chat.docs.StaticInstances.TAG
+import java.io.File
+
+object Messages {
+    val client: ChatClient = TODO()
```

Comment:
What's about providing these dependencies in the constructor as arguments?

Reply:
The same in Java snippets

Reply:
ah it's better than `TODO()` 😬